initial start on assistant docs (#201)
beaverbuilder/documentation: docs/assistant/plugin/apps/libraries.md

1. Open the **Libraries App**.
2. Navigate to the library you want to add color(s) to.
3. Click the **Plus** (+) icon and select **Colors**.
4. Select your color from the color picker or enter numeric values to select a color.
5. Name your color (by default the name will be the color’s hex code.).

![Add a Color to a library](/img/assistant/cloud--libraries--assistant-plugin--3.jpg)

### Add Theme Settings

This procedure saves your site’s customizer settings.

1. Open the **Libraries App**.
2. Navigate to the library you want to add the theme settings to.
3. Click the **Plus** (+) icon.
4. Select **Settings**.
5. Click **Add Settings**.

![Add a Theme Settings to a library](/img/assistant/cloud--libraries--assistant-plugin--4.jpg)

## Delete a Library

To delete a library from the **Libraries App**:

1. Open the **Libraries App**.
2. Select the library you want to delete.
3. Click the **Settings** icon.
4. Scroll down to the **Danger Zone** section.
5. Click the **Delete Library** button in the **Delete this library** section.
6. Confirm the deletion.

![Delete library from Assistant plugin](/img/assistant/cloud--libraries--delete-library--2.jpg)

## Launch Assistant Pro Cloud

> The Assistant Pro Cloud can be launched from within the **Libraries App** providing quick access. To launch the Assistant Pro Cloud from the **Libraries App**:

Reviewer: ```suggestion
You can quickly launch Assistant Pro Cloud in a browser from within Assistant plugin's **Libraries App**. 

To launch Assistant Pro Cloud from the **Libraries App**:
```

1. Open the **Libraries App**.

> 2. Click your avatar to deploy the dropdown menu.

Reviewer: ```suggestion
2. Click your avatar in the upper right corner.
```
```suggestion
2. Click your avatar to deploy the dropdown menu.
```

---
id: libraries
title: Libraries App
sidebar_label: Libraries
---

> The Libraries App is where you connect your WordPress sites to Assistant Pro so you can add and access assets to and from the Assistant Pro Cloud. 

Reviewer 2: You connect multiple sites.

For in depth information on how to add content to the Assistant Pro Cloud from the Libraries App, see the help resources on How to ​​Create a New Library From the Assistant WordPress Plugin and Adding Content to a Library From the Libraries App in the Assistant Pro Cloud Overview Section of this documentation. To learn how to access content from the Assistant Pro Cloud, see the section on Using Assistant Pro Library Assets On Sites.

![Libraries app overview](/img/assistant/apps--apps-libraries--1.jpg)

## Launch Assistant Pro Cloud

> There is a link to the Assistant Pro Cloud in the Libraries App of the WordPress Plugin. If you are connected to Assistant Pro, from the Libraries App you will see your user avatar. Click your avatar and select Launch Cloud from the dropdown menu.

Reviewer 2: This is a common misconception in Nancy's notes... there is only one plugin avalible.  There are multiple levels of Cloud user accounts.  Maybe this is something that needs to be clarified so people understand?
